pkg/simple/client/k8s/kubernetes.go

```diff
@@ -8,6 +8,7 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	kubesphere "kubesphere.io/kubesphere/pkg/client/clientset/versioned"
+	snapshot "kubesphere.io/kubesphere/pkg/client/clientset/versioned"
```

Comment:
Little confused here, why we need to import snapshot independently?

Reply:
It's useless, I'ill delete this.